src/platform/Darwin/DnssdImpl.cpp

```diff
@@ -26,15 +26,19 @@
 #include <lib/support/logging/CHIPLogging.h>
 #include <platform/CHIPDeviceLayer.h>
 
+using namespace chip;
 using namespace chip::Dnssd;
 using namespace chip::Dnssd::Internal;
 
 namespace {
 
-constexpr char kLocalDot[] = "local.";
+// The extra time in milliseconds that we will wait for the resolution on the open thread domain to complete.
+constexpr uint16_t kOpenThreadTimeoutInMsec = 250;
+
+static bool hasOpenThreadTimerStarted = false;
```

Comment:
Shouldn't this be part of the `ResolveContext` rather than static?

Reply:
It absolutely needs to be.

Reply:
done.